standalone/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ if(NOT TARGET ttkUserInterfaceBase)
   set(gui_dirs "")
 endif()
 
+add_compile_options(${TTK_COMPILER_FLAGS})
```

Comment:
Using the non targeted version is deprecated, but honestly for the standalone if its avoid modifying each files I can accept this solution.

Reply:
Yeah, this was the only way I found to do it in a minimum number of lines (too many files to modify otherwise). I agree that this is not really following best practices.

Reply:
that'd be nice to add a comment here, like:
```
# warning deprecated
```

Reply:
I've added a warning on the `add_compile_option` call.